tests/network/expose.go

```diff
@@ -572,7 +563,7 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		var vmrs *v1.VirtualMachineInstanceReplicaSet
 		BeforeEach(func() {
 			By("Creating a VMRS object with 2 replicas")
-			template := newLabeledVMI("vmirs", virtClient, false)
+			template := newLabeledVMI("vmirs", virtClient)
```

Comment:
Maybe we can inline this below.

Reply:
I'm not troubled by temporary variables, but if you think it should go - it's gone.